styles/style.css

```diff
@@ -0,0 +1,172 @@
+/*body {
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+}
+header,
+footer {
+    background-color: #f8f8f8;
+    padding: 10px 20px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+header nav a,
+footer div a {
+    margin: 0 10px;
+    text-decoration: none;
+    color: #333;
+}
+.container {
+    padding: 20px;
+}
+.section-title {
+    margin: 20px 0;
+}
+.grid {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 20px;
+}
+.grid-item {
+    flex: 1 1 calc(33.333% - 20px);
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+    padding: 20px;
+    box-sizing: border-box;
+}
+.grid-item.large {
+    flex: 1 1 calc(50% - 20px);
+}
+.grid-item img {
+    width: 100%;
+    height: auto;
+}
+.grid-item h3,
+.grid-item p,
+.grid-item .price {
+    margin: 10px 0;
+}
+.buy-now {
+    display: block;
+    text-align: center;
+    background-color: #28a745;
+    color: white;
+    padding: 10px;
+    text-decoration: none;
+}
+.add-ons,
+.footer-content {
+    display: flex;
+    justify-content: space-between;
+}
+.footer-content div {
+    flex: 1;
+}
+.newsletter input[type="text"] {
+    padding: 5px;
+    width: 70%;
+}
+.newsletter input[type="submit"] {
+    padding: 5px;
+} */
+
+/* rides.html */
+
+.map {
+    width: 100%;
+    height: auto;
+}
+
+.jurassic-ride h2 {
+    color: orange;
+    text-align: center;
+    font-size: 120px;
+}
+
+.jurassic-images {
+    display: flex;
+    justify-content: center;
+    gap: 5px;
+}
+
+.rides-desc {
+    text-align: center;
+}
+
+.jurassic-buttons {
+    display: flex;
+    justify-content: center;
+    gap: 15px;
+}
+
+
+.fantasy-land h2 {
+    color: green;
+    text-align: center;
+    font-size: 120px;
+}
+
+.fantasy-images {
+    display: flex;
+    justify-content: center;
+    gap: 5px;
+}
+
+.fantasy-buttons {
+    display: flex;
+    justify-content: center;
+    gap: 15px;
+}
+
+.future-world h2 {
+    color: blue;
+    text-align: center;
+    font-size: 120px;
+}
+
+.future-images {
+    display: flex;
+    justify-content: center;
+    gap: 5px;
+}
+
+.future-buttons {
+    display: flex;
+    justify-content: center;
+    gap: 15px;
+}
+
+.wild-west h2 {
+    color: rgb(248, 97, 37);
+    text-align: center;
+    font-size: 120px;
+}
+
+.wild-images {
+    display: flex;
+    justify-content: center;
+    gap: 5px;
+}
+
+.wild-buttons {
+    display: flex;
+    justify-content: center;
+    gap: 15px;
+}
+
+.roller-coaster {
+    color: black;
+    text-align: center;
+    font-size: 75px;
+}
+
+.rides-desc {
+    color: black;
+    text-align: center;
+    font-size: xx-large;
+}
+
+.round-button {
+    border: 2px;
+    border-radius: 15px;
+}
```

Comment:
I suggest keeping a css file per html, so we would have "pairs" and it'll be more clear.
You just need to create a new `rides.css` and insert that css there. Also need referencing it in the main [`index.html` as done here](https://github.com/gabriel-viviani/uol-cm-1040/blob/main/index.html#L7)